plugins/custom-class/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta charset="utf-8" />
+<link rel="shortcut icon" href="favicon.png" />
+<title>Custom Class ▲ Prism plugins</title>
+<base href="../.." />
+<link rel="stylesheet" href="style.css" />
+<link rel="stylesheet" href="themes/prism.css" data-noprefix />
+<script src="prefixfree.min.js"></script>
+<script>var _gaq = [['_setAccount', 'UA-33746269-1'], ['_trackPageview']];</script>
+<script src="http://www.google-analytics.com/ga.js" async></script>
+</head>
+<body class="language-javascript">
+
+<header>
+	<div class="intro" data-src="templates/header-plugins.html" data-type="text/html"></div>
+
+	<h2>Custom Class</h2>
+	<p>This plugin allows you to prefix Prism default classes (<code>.comment</code> will become <code>.namespace--comment</code>) or replace them with your defined ones (like <code>.editor__comment</code> or <code>.comment_7sh3a</code>).</p>  
+</header>
+
+<section>
+	<h1>Motivation</h1>
+
+	<p>Prism default classes are sensible but fixed and too generic. This plugin provide some ways to customize those classes to suite your needs. Example usages:</p>
+
+	<ul>
+		<li>
+			 You want to add namespace for all of them (like <code>.prism--comment</code>) to avoid conflict with your existed classes.
+		</li>
+		<li>
+			You use a naming convention (like <a href="https://en.bem.info/method">BEM</a>). You want to write classes like <code>.editor__comment</code>.
+		</li>
+		<li>You use
+			<a href="https://github.com/css-modules/css-modules">CSS Modules</a>. You want to use your hashed classes, like <code>.comment_7sh3a</code>.
+		</li>
+	</ul>
+
+	<h1>Features</h1>
+
+	<p>This plugin currently provides 2 features:</p>
+
+	<h2>1. Prefix all Prism classes with a string</h2>
+
+	<code>Prism.plugins.customClass.prefix('prism--')</code>
+
+	<h2>2. Replace some Prism classes with your defined ones via an object</h2>
+
+	<pre class="language-js"><code>Prism.plugins.customClass.map({
+	keyword: 'special-keyword',
+	string: 'string_ch29s',
+	comment: 'comment_93jsa'
+})</code></pre>
+
+	<p>Object's keys are the tokens you want to replace (eg: <code>comment</code>), with its value is the classes you want to use (eg: <code>my-comment</code>). Tokens which are not specified will stay the same.</p>
```

Comment:
"with its value is the" should probably be replaced with something like... "with their values being the", I guess...